### ISSUE TITLE
Enable different versioning

### ISSUE THREAD

**Issue by kosciCZ** [user-cont/release-bot#12]
> the PR must be named in this format: `0.1.0 release`

Enable user defined names and version formats

Update: consider implementing a new way of triggering a release:
* keyword "new major release" -- the bot would bump x
* keyword "new minor release" -- the bot would bump y
* keyword "new patch release" -- the bot would bump z

This is equivalent to the [Semantic versioning](https://semver.org/) specification.
Example: latest version is 1.4.2 (x=1, y=4, z=2):
* "new major release" would resulted into version 2.0.0
* "new minor release" would resulted into version 1.5.0
* "new patch release" would resulted into version 1.4.3

Update2: since not every project uses semver, the bot should be able to work in such environment.

**Comment by TomasTomecek** [comment]
Related #117

**Comment by shresthagrawal** [comment]
Hello,
What does x,y,z refers too in the above issue. can you provide a more elaborate example.

**Comment by TomasTomecek** [comment]
I updated, I hope it's more clear now.

**Comment by shresthagrawal** [comment]
Yaah it is clear now. Thanks. I am working on this issue.

**Comment by TomasTomecek** [comment]
I hang out on freenode as ttomecek, but am available only while at work (8:00 - 16:00 UTC)

**Comment by shresthagrawal** [comment]
Hey, just a small question. Is their any function to get the current latest version of the release so thay I can perform the increment on it. Second if their are no release what should be the first release

**Comment by TomasTomecek** [comment]
We discussed on IRC, my replies:
```
15:53 <ttomecek> so we have two methods: for both PyPI and Github releases:
15:53 <ttomecek> https://github.com/user-cont/release-bot/blob/b6f06f1f65bacb4915a63cbbc46e9b0db97bdd04/release_bot/github.py#L196
15:53 <ttomecek> https://github.com/user-cont/release-bot/blob/02237d8f816e8d0220b9eb1794c99b6ec87226b1/release_bot/pypi.py#L38
15:54 <arash> Okay and if their is no release in that case what should I do?
15:55 <arash> Should the release start with 0.0.0 by default
15:56 <arash> Are the pypi version and GitHub versions not the same ?
15:58 <ttomecek> well, they should be the same, but there is possibility they won't be; I'd probably get the latest github release
16:00 <ttomecek> the problem is that the issue covers two different tasks 
16:01 <ttomecek> but if we're talking about the "new major/minor release" workflow: if there is no release, I'd expect the latest version to be 0.0.0 as you suggests
```

**Comment by shresthagrawal** [comment]
@TomasTomecek can you please close this issue

**Comment by rpitonak** [comment]
Solved by #164

**Comment by jpopelka** [comment]
> Solved by #164

are you sure? (#164 was closed)

**Comment by rpitonak** [comment]
@jpopelka thanks, should be fixed now
